randovania/games/dread/json_data/Artaria.json

```diff
@@ -5376,8 +5376,42 @@
                     "editable": true,
                     "connections": {
                         "Event - Post Chain Reaction Debris": {
-                            "type": "template",
-                            "data": "Shoot Diffusion or Wave"
```

Comment:
We'll want another node with the same event - one for destroying from above/the teleport, and one for destroying from the water.

As it is, logic thinks if you destroy the blob from below, then the blob's only connection is to the teleport, so it thinks you can go from water -> blob -> teleport, without morph ball.